features/proposed/add-blocklist/README.md

```diff
@@ -0,0 +1,64 @@
+# Integrate Botnet Controller List (BCL)
```

Comment:
Let's rename this to be less about the specific list "BCL" and more about consuming blocklists in general.